client/elements/FluxFail.jsx

```diff
@@ -71,8 +72,6 @@ class FluxFail extends React.Component {
           view={this.props.view}
           onNavigate={this.props.onNavigate}
           onLogout={this.props.onLogout}
-          allowAddDelay={allowAddDelay}
```

Comment:
Ich glaub, das `allowAddDelay` muss drin bleiben, sonst kriegt man den Button auch dann, wenn man garnicht eingeloggt ist. Siehe dazu:
- https://github.com/FluxFail/flux.fail/blob/feature/public-beta-prelaunch/client/elements/FluxFail.jsx#L18-L22

Reply:
Der runde Add-Button rechts unten wurde wie gefordert aus der Navigation-Component vollständig entfernt. Somit führt aktuell die einzige Möglichkeit zum Hinzufügen von Delays über den neuen Add-delay-Button in der DelayList und diese wiederum ist laut FluxFail.jsx z45-z63 nur mit einem token erreichbar. Das Attribut allowAddDelay auf das du verweist, wird zwar in FluxFail.jsx gesetzt, anschließend allerdings in keiner Logik verwendet.